VideoWeb/VideoWeb/Extensions/ConfigureAuthSchemeExtensions.cs

```diff
@@ -110,16 +110,16 @@ private static void AddPolicies(AuthorizationOptions options, IList<IProviderSch
             var allRoles = new[]
             {
                 AppRoles.CitizenRole, AppRoles.JudgeRole, AppRoles.RepresentativeRole, AppRoles.CaseAdminRole,
-                AppRoles.VhOfficerRole, AppRoles.JudicialOfficeHolderRole, AppRoles.StaffMemberRole
+                AppRoles.VhOfficerRole, AppRoles.JudicialOfficeHolderRole, AppRoles.StaffMember
             };
 
             var rolePolicies = new Dictionary<string, string[]>
             {
                 [AppRoles.JudgeRole] = new[] { AppRoles.JudgeRole },
                 [AppRoles.VhOfficerRole] = new[] { AppRoles.VhOfficerRole },
-                ["Judicial"] = new[] { AppRoles.JudgeRole, AppRoles.JudicialOfficeHolderRole, AppRoles.StaffMemberRole },
+                ["Judicial"] = new[] { AppRoles.JudgeRole, AppRoles.JudicialOfficeHolderRole, AppRoles.StaffMember },
```

Comment:
Although this has already been like this, I think to keep consistent we should declare this as consts similarly to the other AppRoles